docker/release/docker-entrypoint.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+set -e
+
+PYTHONPATH="/app"
+
+# Pass all args to dftimewolf
+if [[ "$1" =~ 'dftimewolf' ]]; then
+    poetry run python -m dftimewolf.cli.dftimewolf_recipes "${@:2}"
```

Comment:
Not sure what you prefer here but you can probably just get away with ```petry run dftimewolf "${@:2}"``` since poetry should install the dftimewolf entrypoint defined in pyproject.toml.

Reply:
yup, that works! much better, thanks